Task4.py

```diff
@@ -0,0 +1,33 @@
+# Multi-paradigm programming languages, Task №4
+# Andrey Fortus IKM-221a
+
+GENERAL_INFO = 'Multi-paradigm programming languages, Task №4'
+STUDENT_INFO = 'Andrey Fortus IKM-221a'
+
+# info
+print(GENERAL_INFO)
+print(STUDENT_INFO)
+
+
+# function for Fibonacci numbers
+def fibonacci(n):
+    if n == 0:
+        return 0
+    elif n < 3:
+        return 1
+    else:
+        return fibonacci(n-1) + fibonacci(n-2)
+
+
+# print n is the number of Fibonacci numbers
+try:
+    n = int(input('Input number of Fibonacci numbers: '))
+except ValueError:
+    print('Invalid value! Try again!')
```

Comment:
I would recommend you **raise** an error instead of printing.
in such case I don't need the next **else** at the line 27